index.js

```diff
@@ -70,8 +61,14 @@ module.exports = (input, opts) => {
 				task: () => del('node_modules')
 			},
 			{
-				title: `Installing dependencies${opts.yarn ? ' using Yarn' : ''}`,
-				task: install(bin, opts.yarn)
+				title: 'Installing dependencies using Yarn',
+				enabled: () => opts.yarn === true,
+				task: () => exec('yarn', ['install'])
```

Comment:
Also, to further remark, this was implemented in https://github.com/sindresorhus/np/pull/114/commits/17091a0e15e76822b12652c38a3c271ba62815c7 but removed because of these concerns.